File: app/src/main/java/com/FaceLab/FaceLab/ApplicationClass.java
package com.FaceLab.FaceLab;

import android.app.Application;
import android.content.Context;
import android.content.SharedPreferences;

/**
 * Created by jionchu on 2021-01-28
 */
public class ApplicationClass extends Application {
    public static SharedPreferences sSharedPreferences = null;

    // SharedPreferences 키 값
    public static String TAG = "FACE_LAB";

    @Override
    public void onCreate() {
        super.onCreate();

        if (sSharedPreferences == null) {
            sSharedPreferences = getApplicationContext().getSharedPreferences(TAG, Context.MODE_PRIVATE);
        }
    }
}

File: app/src/main/java/com/FaceLab/FaceLab/ApplicationClass.kt
package com.FaceLab.FaceLab

import android.app.Application
import android.content.SharedPreferences

/**
 * Created by jionchu on 2021-01-28
 */
class ApplicationClass : Application() {
    override fun onCreate() {
        super.onCreate()
        if (sSharedPreferences == null) {
            sSharedPreferences = applicationContext.getSharedPreferences(TAG, MODE_PRIVATE)
        }
    }

    companion object {
        var sSharedPreferences: SharedPreferences? = null

        // SharedPreferences 키 값
        var TAG = "FACE_LAB"
    }
}